lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
```

Comment:
I think that it's okay usually in my world it is, but this is a judgement call - I'm not mad about it :D